client/app/services/dashboard.js

```diff
@@ -164,6 +164,7 @@ const DashboardService = {
     return axios.get(`api/dashboards/${id || slug}`, { params }).then(transformResponse);
   },
   getByToken: ({ token }) => axios.get(`api/dashboards/public/${token}`).then(transformResponse),
+  copy: ({ slug }) => axios.post("api/dashboards/duplicate", { slug }).then(transformResponse),
```

Comment:
Let's keep the same naming convention as for queries: use `fork` instead of `copy`/`duplicate` (here and in other places)

Reply:
I fixed all your comments!